CMake/FindLuaJit.cmake

```diff
@@ -0,0 +1,43 @@
+# from https://github.com/neovim/neovim/blob/master/cmake/FindLuaJit.cmake
+# - Try to find luajit
+# Once done this will define
+#  LUAJIT_FOUND - System has luajit
+#  LUAJIT_INCLUDE_DIRS - The luajit include directories
+#  LUAJIT_LIBRARIES - The libraries needed to use luajit
+#
+# Copyright neovim, Apache 2.0 license.
```

Comment:
Apache 2.0 is not compatible with GPLv2+ (which this project is). We cannot use this until you contact the author and get an explicit exception for using this.

Reply:
Hm, this is unfortunate, I'll reach out to them. In any case is it fine linking LuaJIT dynamically through pkg-config like this? Or should it be moved to `Externals` like some of the other deps?

Reply:
FindLuaJit.cmake was removed so this is resolved.